src/Approvals/SFA.DAS.Approvals/InnerApi/Responses/GetBulkUploadUpdateLogResponse.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SFA.DAS.Approvals.InnerApi.Responses
+{
+    public class GetBulkUploadUpdateLogResponse
+    {
+        public int LogId { get; set; }
+    }
+}
```

Comment:
sorry, see previous comments about unused code

Reply:
hi paul, removed this as part of removing all Update endpoint